### DISCUSSION TITLE
【请教】engine 开源代码是否包含 native runtime 部分？Alipay 小程序对接 oasis 的底层方式是什么？

### DISCUSSION THREAD

**Discussion by kenkozheng** [galacean/discussion#60] [Mini Program（小程序）] [answered]
感谢 oasis 团队给大家带来这个高效引擎。

小弟学习之中有一些疑问。
engine 开源代码是否包含 native runtime 部分？Alipay 小程序对接 oasis 的底层方式是什么？

**Comment by gz65555** [comment, marked as answer]
1. 没包含 native runtime 的部分。
2. 只是小程序底层暴露对应的标准 WebGL API 提供给 Oasis 进行对接，然后 Oasis 的 js 运行环境对接到小程序的运行环境。
3. 后面会有文章去做解析。

**Comment by gz65555** [comment]
@kenkozheng 有问题可以继续问，这个 issue 先关了。
